src-tauri/src/lib.rs

```diff
@@ -1,15 +1,62 @@
+use std::collections::HashMap as Map;
 // Learn more about Tauri commands at https://tauri.app/v1/guides/features/command
+#[derive(Debug, serde::Deserialize)]
+struct Currency {
+    conversionRate: f32,
+    existingAmount: f32,
+}
```

Comment:
Nitpick: better to rename this struct into Coin or Money or something else, and bring another struct Currency keeping only currency name (short text code like USD or ATOM) and auxiliary data like country and full_name. Later we'll need text search by these fields.

I think, exchange rates also should be kept separately in memory because they change often, but the currency metadata doesn't change, e.g. USD is always U.S. dollar.